server/model_server/utils.py

```diff
@@ -10,3 +10,25 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         sys.stdout.close()
         sys.stdout = self._original_stdout
+
+# https://medium.com/pythonhive/python-decorator-to-measure-the-execution-time-of-methods-fa04cb6bb36d
+def timeit(method):
+    """a decorator for testing excution time like colde start
+    
+    Arguments:
+        method {[type]} -- [description]
+    
+    Returns:
+        [type] -- [description]
+    """
+    def timed(*args, **kw):
+        ts = time.time()
+        result = method(*args, **kw)
+        te = time.time()
+        if 'log_time' in kw:
+            name = kw.get('log_name', method.__name__.upper())
+            kw['log_time'][name] = int((te - ts) * 1000)
+        else:
+            print('%r  %2.2f ms' % (method.__name__, (te - ts) * 1000))
```

Comment:
Can we use a Logger? I believe there are some other outputs when running functions (e.g. gRPC call), use a Logger to output the result directly into a file would be nicer.

Reply:
Also use the new format style would be better.
```suggestion
            print(f'{method.__name__!r}  {(te - ts) * 1000:2.2f} ms')
```

Reply:
```python
def timeit(method):
    """Use to time a running time of a method.
    Adapted from https://medium.com/pythonhive/python-decorator-to-measure-the-execution-time-of-methods-fa04cb6bb36d
    """
    def timed(*args, **kwargs):
        # obtain logger
        _log_time = kwargs.pop('_log_time', None)
        _log_name = kwargs.pop('_log_name', method.__name__.upper())
        
        ts = time.time()
        result = method(*args, **kwargs)
        te = time.time()
        if _log_time:
            kwargs['__log_time'][_log_name] = int((te - ts) * 1000)
        else:
            print(f'{method.__name__!r}  {(te - ts) * 1000:2.2f} ms')
        return result
    return timed
```